projects/vdk-core/src/vdk/api/plugin/plugin_input.py

```diff
@@ -237,34 +238,43 @@ def ingest_payload(
         destination_table: Optional[str],
         target: Optional[str] = None,
         collection_id: Optional[str] = None,
+        metadata: Optional[IngestionMetadata] = None,
```

Comment:
Optional[IngestionMetadata] seems correct; should propagate throughout entire chain, from pre- to post- processing

Reply:
Gotcha. Ok, thanks